templates/base.tpl

```diff
@@ -5,19 +5,19 @@
 
 <script type="text/javascript">
     "use strict";
-    let goecharts_{{ .ChartID | safeJS }} = echarts.init(document.getElementById('{{ .ChartID | safeJS }}'), "{{ .Theme }}", { renderer: "{{ .Renderer }}" });
+    let goecharts_{{ .ChartID | safeJS }} = echarts.init(document.getElementById('{{ .ChartID | safeJS }}'), "{{ .Theme }}", { renderer: "{{  .Initialization.Renderer }}" });
```

Comment:
Ah, thanks for this one.

Reply:
In fact, even with this fix it does not work for some reason (anymore?). The `.Initialization` struct seems to be the default one always. Even if I set `page.Initialization.Renderer = "svg"` it does not change what is written out here. And other values, like `page.PageTitle` which I do set, is also not available here, just default values.

Reply:
> In fact, even with this fix it does not work for some reason (anymore?). The `.Initialization` struct seems to be the default one always. Even if I set `page.Initialization.Renderer = "svg"` it does not change what is written out here. And other values, like `page.PageTitle` which I do set, is also not available here, just default values.

I test it locally, it works as expect. try this?
```go
func barBasic() *charts.Bar {
	bar := charts.NewBar()
	bar.Initialization.Renderer = "svc"
	// ...

	return bar
}

```
About the Page config, I guess it may have some issue exist, plz raise a issue let us trace it.